fic_modules/git.py

```diff
@@ -224,7 +239,8 @@ def filter_git_scriptworkers(latest_releases, repo_team, script_repo):
         if last_modified <= new_commit_version_date:
             each_commit2 = {}
             commit_number += 1
-            each_commit2.update({int(commit_number): get_commit_details(commit2)})
+            each_commit2 \
```

Comment:
Very weird break.
Please put a new line after `.update({`

Reply:
Modified into this:
``` Python
commit_details = get_commit_details(commit2)
each_commit2.update({int(commit_number): commit_details})
```

Reply:
If everything looks good after run it will be added to commit.